gradle-consistent-versions-idea-plugin/src/main/java/com/palantir/gradle/versions/intellij/RepositoryExplorer.java

```diff
@@ -33,12 +36,24 @@ public class RepositoryExplorer {
     private static final Logger log = LoggerFactory.getLogger(RepositoryExplorer.class);
 
     private final String baseUrl;
+    private static final Cache<DependencyGroup, List<Folder>> folderCache;
```

Comment:
nit: I usually initialize the static fields directly in the declaration, makes it easy to read, esp. because the initialization logic is simple

Reply:
good to know thanks!